plugins/foreman_openscap/1.0/index.md

```diff
@@ -0,0 +1,574 @@
+
+
+## 1.2 Release notes
+
+Compatibility matrix
+
+<table class="table table-bordered table-condensed">
+  <tr>
+    <th>Foreman version</th>
+    <th>Plugin version</th>
+    <th>Proxy version</th>
+    <th>Client version</th>
+  </tr>
+  <tr>
+    <td>>= 1.17</td>
+    <td>0.9.0</td>
+    <td>>=0.6.0</td>
+    <td>0.3.0</td>
+  </tr>
+  <tr>
+    <td>1.22</td>
+    <td>1.0.1</td>
+    <td>0.7.x</td>
+    <td>0.4.6</td>
+  </tr>
+  <tr>
+    <td>1.23</td>
+    <td>1.0.9</td>
+    <td>0.7.x</td>
+    <td>0.4.6</td>
+  </tr>
+</table>
+
+# 2. Installation
+
+There are a few components to install:
+
+* Foreman OpenSCAP (foreman_openscap)
+* Smart Proxy OpenSCAP (smart_proxy_openscap)
+* puppet-foreman_scap_client or ansible-foreman_scap_client
+
+## 2.1 Installing foreman_openscap
+
+The easiest and recommended way is using foreman-installer. It's as easy as running this command
+
+	> foreman-installer --enable-foreman-plugin-openscap
+
+If you prefer the manual way, you can install it from package like this:
+
+	> yum install tfm-rubygem-foreman_openscap
+	> foreman-rake db:migrate
+	> foreman-rake db:seed
+	> service httpd reload
+
+Please refer to the Foreman plugin [manual]({{site.baseurl}}/manuals/latest/index.html#6.1InstallaPlugin) for more information about installing Foreman plugins.
+
+## 2.2 Installing smart_proxy_openscap
+
+The easiest and recommended way is using foreman-installer. It's as easy as running this command
+
+	> foreman-installer --enable-foreman-proxy-plugin-openscap
+
+If you run Smart-Proxy with Foreman on the same host you can combine it with --enable-foreman-plugin-openscap option.
+
+If you prefer the manual way, install this package on the Smart-Proxy server:
+
+    > yum install rubygem-smart_proxy_openscap
+
+Edit ```/etc/foreman-proxy/settings.d/openscap.yml``` with the appropriate settings
+```yaml
+    ---
+    :enabled: true
+
+    # Log file for the forwarding script.
+    :openscap_send_log_file: /var/log/foreman-proxy/openscap-send.log
+
+    # Directory where OpenSCAP audits are stored
+    # before they are forwarded to Foreman
+    :spooldir: /var/spool/foreman-proxy/openscap
+
+    # Directory where OpenSCAP content XML are stored
+    # So we will not request the XML from Foreman each time
+    :contentdir: /var/lib/openscap/content
+
+    # Directory where OpenSCAP report XML are stored
+    # So Foreman can request arf xml reports
+    :reportsdir: /usr/share/foreman-proxy/openscap/reports
+
+    # Directory where OpenSCAP report XML are stored
+    # In case sending to Foreman succeeded, yet failed to save to reportsdir
+    :failed_dir: /usr/share/foreman-proxy/openscap/failed
+
+    # Directory where corrupted OpenSCAP report XML are stored
+    # When proxy cannot parse the report sent by client
+    :corrupted_dir: /var/lib/foreman-proxy/openscap/corrupted
+
+    # Timeout for sending OpenSCAP reports to Foreman server, in seconds
+    :timeout: 60
+```
```

Comment:
This should be unindented so it can be copy-pasted (yaml is very sensitive to spaces)